package.json

```
@@ -64,10 +64,10 @@
     "prettier": "1.18.2",
     "semantic-release": "^15.13.24",
     "ts-jest": "^25.3.0",
-    "typescript": "^3.8.3"
+    "typescript": "^4.0.3"
   },
   "peerDependencies": {
-    "eslint": "^5 || ^6"
+    "eslint": "^7.5.0"
```

Same as last comment: per previous discussion this should be any ESLint version greater or equal than 5
```suggestion
    "eslint": ">=5.0.0"
```

---

I did this on purpose, since `@typescript-eslint/experimental-utils@4.x` requires `eslint@^7.5.0` in order to have the optional chaining support

---

Not sure about this. This will force all users to migrate their ESLint to a version >= 7.5 and that's a huge limitation. As per private discussion, CRA is doing so, so probably we are fine?

---

CRA is doing so indeed and it's needed for optional chaining support too.